server/config.go

```diff
@@ -674,6 +674,7 @@ type SessionConfig struct {
 	TokenExpirySec        int64  `yaml:"token_expiry_sec" json:"token_expiry_sec" usage:"Token expiry in seconds."`
 	RefreshEncryptionKey  string `yaml:"refresh_encryption_key" json:"refresh_encryption_key" usage:"The encryption key used to produce the client refresh token."`
 	RefreshTokenExpirySec int64  `yaml:"refresh_token_expiry_sec" json:"refresh_token_expiry_sec" usage:"Refresh token expiry in seconds."`
+	RefreshTokenRotation  bool   `yaml:"refresh_token_rotation" json:"refresh_token_rotation" usage:"Enable refresh token rotation. Default false."`
```

Comment:
We think it'd be best if this defaults to `true`.

Reply:
When we enable this feature on server side we should disable session auto-refresh on client side. Because auto-refresh on client side can leads to token refresh race condition (for example client calling two RPC on same time - two simultaneous calls to refresh token as result)

Reply:
if you feel ok with this, i will set defaults to true

Reply:
previously refresh token was immutable once created, so clients could persist it on login once and then ignore returned refresh token  from the refresh session api call. if we turn it on by default this change should be listed prominently in release notes as potential breaking change.

Reply:
@redbaron in the end we decided not to turn it on by default as this requires some coordination with our SDKs release and as you mention is a breaking change.